scripts/update-size.test.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const root = path.join(__dirname, '..');
+const pkg = require(path.join(root, 'package.json'));
+const file = path.join(root, pkg.browser[pkg.module]);
+const size = require('./update-size');
+
+describe('size-limit', () => {
+  it('check size to be object', () => {
+    return size
+      .sizeLimit(file, { ignore: ['react', 'prop-types'] })
+      .then(result => {
+        expect(result).toBeInstanceOf(Object);
```

Comment:
this test doesnt really test anything - we dont want to retest size-limit package and its API